### DISCUSSION TITLE
Tooltip

### DISCUSSION THREAD

**Discussion by heesters** [sgratzl/chartjs-chart-boxplot#154] [Q&A]
Can anyone help me with this probably simple problem?
Everything works as it should, I just want the tooltip to only provide the mean and the median and preferably on 2 lines.

**Comment by sgratzl** [comment]
take a look at the normal chart.js way to customize the tooltip. This plugin actually enhances the data elements that you ar getting such that you have access to all details.

**Reply by heesters** [reply]
Although good advice, that is obviously what I did first, that didn't workout so I came here.
I can build an entire new tooltip, but that seems a bit exaggerated when I just want less...
Can you elaborate a bit on what your callbacks are? The documentation is not very clear on this.

**Reply by heesters** [reply]
tooltip: {
        callbacks: {
          label: function(context) {
            const dataset = context.dataset;
            const dataIndex = context.dataIndex;
            const median = dataset.data[dataIndex].median;
            const mean = dataset.data[dataIndex].mean;
            return [
              `Median: ${median.toFixed(2)}`,
              `Mean: ${mean.toFixed(2)}`
            ];
          },
          On the right track?
